tests/test_rootfs.py

```diff
@@ -94,15 +94,20 @@ def test_expected_files_ext234(
             output = subprocess.check_output(
                 ["debugfs", "-R", "ls -l -p /data", latest_rootfs], cwd=tmpdir
             ).decode()
+            at_least_one = False
             for line in output.split("\n"):
                 splitted = line.split("/")
                 if len(splitted) <= 1:
                     continue
+
+                at_least_one = True
+
                 # Should only contain "." and "..". In addition, debugfs
                 # sometimes, but not always, returns a strange 0 entry, with
                 # no name, but a "0" in the sixth column. It is not present
                 # when mounting the filesystem.
                 assert splitted[5] == "." or splitted[5] == ".." or splitted[6] == "0"
+            assert at_least_one, "No /data dir on rootfs?"
```

Comment:
This is fine. A more Pythonic way of doing this:
```suggestion
            for line in output.split("\n"):
                splitted = line.split("/")
                if len(splitted) > 1:
                    # Should only contain "." and "..". In addition, debugfs
                    # sometimes, but not always, returns a strange 0 entry, with
                    # no name, but a "0" in the sixth column. It is not present
                    # when mounting the filesystem.
                    assert splitted[5] == "." or splitted[5] == ".." or splitted[6] == "0"
                    break
            else:
                pytest.fail("No /data dir on rootfs?")
```

Reply:
I think it will not work. It needs to go through all entries, but in the suggested version it will break after the first passing one.

Reply:
My bad, I thought it only had to verify one entry.

Looks good.